client/state/index.js

```diff
@@ -111,7 +112,9 @@ export const reducer = combineReducers( {
 	ui,
 	users,
 	wordads,
-} );
+};
+
+export const reducer = combineReducers( { ...extensions.reducers(), ...reducers } );
```

Comment:
we're making a choice here to overwrite extension reducers with the built-ins if an extension uses the same name as a reducer existing already in Calypso. is this the choice we want?